src/function/vector_list_operation.cpp

```diff
@@ -340,9 +344,18 @@ std::unique_ptr<FunctionBindData> ListSortVectorOperation::bindFunc(
     const binder::expression_vector& arguments, FunctionDefinition* definition) {
     auto vectorOperationDefinition = reinterpret_cast<VectorOperationDefinition*>(definition);
     switch (arguments[0]->dataType.getChildType()->getTypeID()) {
+    case INT16: {
+        vectorOperationDefinition->execFunc = getExecFunction<int16_t>(arguments);
+    } break;
+    case INT32: {
+        vectorOperationDefinition->execFunc = getExecFunction<int32_t>(arguments);
+    } break;
     case INT64: {
         vectorOperationDefinition->execFunc = getExecFunction<int64_t>(arguments);
     } break;
+    case FLOAT: {
```

Comment:
Our convension for switch/case ordering is:
case INT64,
case INT32,
case INT16,
case DOUBLE,
case FLOAT,
reorder your case statements